session/src/test/java/org/apache/iotdb/session/IoTDBSessionIT.java

```diff
@@ -622,6 +622,17 @@ public void TestSessionInterfacesWithDisabledWAL()
       tablet.rowSize++;
     }
 
+    for (int time = 101; time <= 200; time++) {
+      int rowIndex = time - 1;
+      tablet.addTimestamp(rowIndex, time);
+      long value = 0;
+      for (int s = 0; s < 3; s++) {
+        tablet.addValue(schemaList.get(s).getMeasurementId(), rowIndex, value);
+        value++;
+      }
+      tablet.rowSize++;
+    }
+
```

Comment:
this test lacks an assert. You can do a select count(*) query and assert

Reply:
i will fix it in PR IOTDB-856:Clear up the IoTDBSessionIT